uberfire-io/src/main/java/org/uberfire/io/impl/AbstractIOService.java

```
@@ -636,10 +636,16 @@ public Path write( final Path path,
 
         try {
             byteChannel.write( ByteBuffer.wrap( bytes ) );
-            byteChannel.close();
         } catch ( final java.io.IOException e ) {
             throw new IOException( e );
         }
+        finally {
+            try {
```

I don't like nested try catch blocks and also silently swallowing an error condition and continuing execution.